Keas.Mvc/ClientApp/components/Access/AccessEditValues.tsx

```diff
@@ -128,7 +128,11 @@ export default class AccessEditValues extends React.Component<IProps, {}> {
         const accessAssignment = this.props.selectedAccess.assignments.filter(
             x => x.personId === personId
         );
-        // TODO: add submitting state
-        await this.props.onRevoke(accessAssignment[0]);
+        try {
+            await this.props.onRevoke(accessAssignment[0]);
+        } catch (err) {
+            // TODO: add submitting state and handle
```

Comment:
@laholstege Just throw an alert in there for now?

Reply:
@jSylvestre it'll still have a toast, but eventually i want to clean assets up and this should have a submitting state